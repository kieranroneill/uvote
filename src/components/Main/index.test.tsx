import { shallow } from 'enzyme';
import * as React from 'react';

// Components.
import { Main } from './';

describe('src/components/Main', () => {
    describe('<Main /> snapshots', () => {
        it('should match the snapshot with default props', () => {
            expect(
                shallow(
                    <Main>
                        <h1>Hello human</h1>
                    </Main>
                )
            ).toMatchSnapshot();
        });
    });
});
