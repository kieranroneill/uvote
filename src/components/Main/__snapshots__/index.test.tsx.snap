// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/Main <Main /> snapshots should match the snapshot with default props 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Main>
    <h1>
      Hello human
    </h1>
  </Main>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <h1>
        Hello human
      </h1>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Hello human",
      },
      "ref": null,
      "rendered": "Hello human",
      "type": "h1",
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <h1>
          Hello human
        </h1>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Hello human",
        },
        "ref": null,
        "rendered": "Hello human",
        "type": "h1",
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
